ledger/test-common/test-certificates/root-ca.cnf.sample

```diff
@@ -0,0 +1,127 @@
+# OpenSSL root CA configuration file.
```

Comment:
Why did we need to switch to a template?

Reply:
I followed the setup provided by Edward Newman in this blog post: https://daml.com/daml-driven/secure-daml-infrastructure---part-1-pki-and-certificates/
The config file allows to reduce duplication because some of the generated certificates in `test-certificates` use the same parameters and values.

Reply:
Sounds good, thanks for the explanation!